test/test_models.py

```diff
@@ -316,6 +316,9 @@ def _check_input_backprop(model, inputs):
     "s3d": {
         "input_shape": (1, 3, 16, 224, 224),
     },
+    "googlenet": {
+        "init_weights": True,
+    },
```

Comment:
Depending on the discussion above, this maybe should be removed again. If we decide to keep `init_weights`, remove the trailing comma to keep this on a single line with our auto formatting:

```suggestion
    "googlenet": {"init_weights": True},
```